internal/runners/export/env.go

```diff
@@ -35,26 +32,15 @@ func NewEnv(prime primeable) *Env {
 func (e *Env) Run() error {
 	rt, _, err := runtime.NewFromProject(e.project, target.TriggerActivate, e.analytics, e.svcModel, e.out, e.auth)
 	if err != nil {
-		if errs.Matches(err, &model.ErrNoMatchingPlatform{}) {
-			branches, err := model.BranchNamesForProjectFiltered(e.project.Owner(), e.project.Name(), e.project.BranchName())
-			if err == nil && len(branches) > 1 {
-				return locale.NewInputError("err_alternate_branches", "", e.project.BranchName(), strings.Join(branches, "\n - "))
-			}
-		}
-		if !authentication.LegacyGet().Authenticated() {
-			return locale.WrapError(err, "err_export_env_auth", "Could not update runtime files. If this is a private project ensure that you are authenticated.")
-		}
-		return locale.WrapError(err, "err_export_new_runtime", "Could not get new runtime")
+		return locale.WrapError(err, "err_export_new_runtime", "Could not initialize new runtime")
 	}
 
 	env, err := rt.Env(false, true)
 	if err != nil {
 		return locale.WrapError(err, "err_env_get_env", "Could not get runtime environment")
 	}
 
-	for k, v := range env {
-		e.out.Print((locale.Tl("env_output_env", "{{.V0}}: {{.V2}}", k, v)))
-	}
+	e.out.Print(env)
```

Comment:
Please make sure you test that this looks as expected with the plain outputter, I don't think we designed the env struct with this in mind (but it might work anyway!).

Reply:
Yep, I've tested it. Here's a sample of the printout:

```
PATH: /Users/mikedrakos/Library/Caches/activestate/05c7ff6d/exec 
PYTHONIOENCODING: utf-8 
PYTHONPATH: /Users/mikedrakos/Library/Caches/activestate/05c7ff6d/_runtime_temp/d0c09e08-e89e-5253-8305-af6770d64992/ActivePython-3.6.6.0000-macosx10.9-x86_64-ae178372/Library/Frameworks/Python.framework/Versions/Current/lib/python3.6/site-packages 
```